src/validators/account-validator.js

```diff
@@ -0,0 +1,231 @@
+const  {check, param}  =  require ( 'express-validator' );
+const {StatusCodes} = require ('http-status-codes');
+const {validateResult} = require('../helpers/response-result');
+
+
+const validationAccount = [
+    check('lastname')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 2, max: 150}).withMessage('El apellido(s) deben tener un mínimo de 2 caracteres y un máximo de 150 caracteres')
+        .matches(/^[a-zA-ZÑñÁáÉéÍíÓóÚúÜü ]+$/).withMessage('Solo se permiten letras y espacios'),
+    check('name')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 2, max: 150}).withMessage('El nombre(s) deben tener un mínimo de 2 caracteres y un máximo de 150 caracteres')
+        .matches(/^[a-zA-ZÑñÁáÉéÍíÓóÚúÜü ]+$/).withMessage('Solo se permiten letras y espacios'),
+    check('age')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .not().isString().withMessage('El campo debe ser un número')
+        .custom((value, { request }) => {
+            if (value < 10 || value > 100) {
+                return false;
+            }
+            return true;
+        }).withMessage('Debe tener más de 10 años y menos de 100 años.'),
+    check('dateBirth')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .matches(/^\d{4}-(0?[1-9]|1[012])-(0?[1-9]|[12][0-9]|3[01])$/).withMessage('La fecha debe tener el formato YYYY-MM-DD')
+        .custom((value, { request }) => {
+            try{
+                const dateBirth = new Date(value).getFullYear();
+                const dateNow = new Date().getFullYear();
+                const year = dateNow- dateBirth;
+                if(year < 10 || year > 100 ){
+                    return false;
+                }
+                return true;
+            }catch(error){
+                return false;
+            }     
+        }).withMessage('El rango de la fecha actual y la fecha de nacimiento debe estar entre 10 y 100 años.')
+]
+
+const validationUsername = [
+    check('username')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 3, max: 20}).withMessage('El nombre de usuario debe tener un mínimo de 3 caracteres y un máximo de 20 caracteres')
+        .matches(/^[A-Za-z0-9]{3,20}$/).withMessage('El nombre de usuario debe tener solo letras y números')
+]
+
+const validationEmail = [
+    check('email')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    check("email")
```